sdk/app_cpu1/common/sys/cmd/cmd_hw.c

```diff
@@ -203,31 +203,35 @@ int cmd_hw(int argc, char **argv)
         if (port < 1 || port > 2)
             return CMD_INVALID_ARGUMENTS;
         else
-            base_addr = EDDY_CURRENT_SENSOR_1_BASE_ADDR;
+            base_addr = TIMING_MANAGER_BASE_ADDR;
 
 #elif USER_CONFIG_HARDWARE_TARGET == AMDC_REV_E
-        if (port == 1)
+        if (port == 1) {
             base_addr = EDDY_CURRENT_SENSOR_1_BASE_ADDR;
-        else if (port == 2)
+            timing_manager_enable_eddy_1();
+        } else if (port == 2) {
             base_addr = EDDY_CURRENT_SENSOR_2_BASE_ADDR;
-        else if (port == 3)
+            timing_manager_enable_eddy_2();
+        } else if (port == 3) {
             base_addr = EDDY_CURRENT_SENSOR_3_BASE_ADDR;
-        else if (port == 4)
+            timing_manager_enable_eddy_3();
+        } else if (port == 4) {
             base_addr = EDDY_CURRENT_SENSOR_4_BASE_ADDR;
-        else
+            timing_manager_enable_eddy_4();
+        } else
             return CMD_INVALID_ARGUMENTS;
 #endif
 
         // hw eddy trigger <port> <HIGH | LOW | BOTH>
         if (argc == 5 && STREQ("trigger", argv[2])) {
-            eddy_current_sensor_trigger_on_pwm_clear(base_addr);
+            timing_manager_trigger_on_pwm_clear();
 
             if (STREQ("HIGH", argv[4]))
-                eddy_current_sensor_trigger_on_pwm_high(base_addr);
+                timing_manager_trigger_on_pwm_high();
             else if (STREQ("LOW", argv[4]))
-                eddy_current_sensor_trigger_on_pwm_low(base_addr);
+                timing_manager_trigger_on_pwm_low();
             else if (STREQ("BOTH", argv[4]))
-                eddy_current_sensor_trigger_on_pwm_both(base_addr);
+                timing_manager_trigger_on_pwm_both();
             else
                 return CMD_INVALID_ARGUMENTS;
 
```

Comment:
Hmm... this should probably be removed. Since the triggering is a "global" level setting, not just for the eddy current sensors. Maybe a new `hw` command which configures the timing manager trigger options?

Reply:
When you remove this, I think it should simplify the code above this part

Reply:
removed this, instead added a new hw command for the timing manager. it currently has enable sensors, set ratio for pwm trigger, and trigger pwm. once the interrupts and therefore cumulative timing stats are working, i'll add one for the user to read the timing